src/backend/core/core/settings.py

```diff
@@ -112,10 +112,10 @@
     # }
     "default": {
         "ENGINE": "django.db.backends.postgresql_psycopg2",
-        "NAME": os.environ.get("POSTGRES_DB", ""),
-        "USER": os.environ.get("POSTGRES_USER", ""),
-        "PASSWORD": os.environ.get("POSTGRES_PASSWORD", ""),
-        "HOST": os.environ.get("POSTGRES_HOST", ""),
+        "NAME": os.environ.get("POSTGRES_DB", "postgres"),
```

Comment:
Could you please set this aj "ecoseller"? Since initial DB in our `docker-compose`is `ecoseller`

Reply:
Done